app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -147,6 +149,16 @@ class FeatureFlagNotificationViewModel @Inject constructor(
         }
     }
 
+    private fun setMLSE2EIdRequiredState(userId: UserId) = viewModelScope.launch {
```

Comment:
following some naming suggestions from [this PR](https://github.com/wireapp/kalium/pull/1832) maybe we can remove the `MLS` from the function name?

just: `setE2EIdRequiredState(..` ?